xbmc/FileItem.h

```diff
@@ -294,6 +296,13 @@ class CFileItem :
     return m_epgInfoTag;
   }
 
+  inline bool HasEPGSearchFilter() const { return m_epgSearchFilter.get() != nullptr; }
```

Comment:
does this pointer need to be unwrapped by .get()? also i think compilers automatically assume `inline` for 1-liner funcs in header files… may be obsolete here, i know it‘s widely used in CFileItem.h

Reply:
> does this pointer need to be unwrapped by .get()? 

Sorry, what do you mean? What code change do you suggest?

Reply:
this :) `bool HasEPGSearchFilter() const { return m_epgSearchFilter != nullptr; }`

Reply:
Ah, yes. Makes perfect sense.